roles/requirements.yml

```diff
@@ -7,8 +7,9 @@
 - src: https://github.com/stackhpc/ansible-role-cluster-infra
   version: newton-backport
   name: stackhpc.cluster-infra
-- src: https://github.com/cedadev/jasmin-cluster-infra-ansible
+- src: https://github.com/stackhpc/jasmin-cluster-infra-ansible
```

Comment:
The `issue-23` PR has been merged, before this commit gets merged can we update this?